### ISSUE TITLE
allErrors not working with multiple forms on the page

### ISSUE THREAD

**Issue by liegeandlief** [ciscoheat/sveltekit-superforms#157]
When I have two forms on a page as follows:

```
	const {
		allErrors: form1AllErrors
	} = superForm(data.form1)
	const {
		allErrors: form2AllErrors
	} = superForm(data.form2)
```

and the forms are initialised on the server with `errors: true`, `$form2AllErrors` contains the errors for form 1.

**Comment by liegeandlief** [comment]
I've just fixed it by specifying IDs for each form when initialising them on the server, but feel this is something which the library could handle internally.

**Comment by ciscoheat** [comment]
Id handling will be pretty much automatic in 1.0, unless the forms are using the same schema, which requires setting the id manually.

**Comment by ciscoheat** [comment]
Cloning this, as it will be fixed in 1.0.
